### PR TITLE
A new era

### PR DESCRIPTION
**Pull request by doomspork**
This is just the start. I'll try to get some of the basics done. 🎉

### CODE REVIEW

**Comment by doomspork** [comment]
@burden how can I move these?
image

File: pages/articles.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+export default function Index({ allPosts }) {
+  return (
+    <>
+      <Layout>
+        <Head>
+          <title>Utensils</title>
+        </Head>
+        <Container>
+        <section className="flex-row flex items-left mt-20">
```
Comment (doomspork) [inline pages/articles.js]: This can all be cleaned up for sure 👍

File: components/post-preview.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export default function PostPreview({
+  title,
+  excerpt,
+  tags,
+}) {
+  return (
+    <div className="p-6 flex flex-col items-start">
+      {tags.map((tag) => (
+        <span className="inline-block py-1 px-2 rounded bg-red-50 text-red-500 text-xs font-medium tracking-widest">{tag}</span>
```
Comment (burden) [inline components/post-preview.js]: Looks like the parent is declaring that it's flex, so you'll want to try `inline-flex` instead of `inline-block`

**Comment by doomspork** [comment]
utensils

**Comment by doomspork** [comment]
Still need to fix the favicon. Also need to correct the ArticleList component so extends wider.

**Comment by doomspork** [comment]
@burden could you take a peek at this whenever you've got a few? I'm okay merging this before the responsiveness if perfected. Hopefully with your and @johnTurknett's help we can clean that part up. Probably do a dark mode too 🎉

File: components/header.js [hunk 3]
```diff
@@ -6,13 +6,15 @@ export default function Header({ subtitle }) {
...
         </a>
       </Link>
-      <a href="/articles" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline mr-4">Articles</a>
-      <a href="https://github.com/utensils" target="_blank" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline">GitHub</a>
+      <span className="inline-block align-middle">
+        <a href="/articles" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline mr-4">Articles</a>
+      </span>
+      <span className="inline-block align-middle">
+        <a href="https://github.com/utensils" target="_blank" className="text-3xl text-red-500 hover:text-red-600 font-black hover:underline">GitHub</a>
+      </span>
```
Comment (doomspork) [inline components/header.js]: @burden / @johnTurknett either of you want to take a peek at how to vertically align these on the logo? I can't seem to get both to do it 🤔

Comment (doomspork) [reply components/header.js:17]: image

This was the best I got 😰

Comment (doomspork) [reply components/header.js]: image

Got it! 🚀

File: components/article-list.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { ar } from 'date-fns/locale'
+import ArticlePreview from './article-preview'
+
+export default function ArticleList({ articles }) {
+  return (
+    <section>
+      <div className="flex flex-rows">
```
Comment (doomspork) [inline components/article-list.js]: image

I'm missing something here.

**Pull request merged**